### PR TITLE
added HW5 with Tasks 1,2,3

### PR DESCRIPTION
**Pull request by azartnyi**
hello
image

### CODE REVIEW

File: HW_5/Task_1.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+// ; 2. Написать скрипт, который выведет 5 строк в консоль таким образом, чтобы в первой строчке выводилось :), во второй :):) и так далее
+// ;   Пример в консоли:
+// ;   :)
+// ;   :):)
+// ;   :):):)
+// ;   :):):):)
+// ;   :):):):):)
+
+const SMILE = ':)';
```
Comment (anatoly-karpovich) [inline HW_5/Task_1.js]:
```suggestion
const smile = ':)';
```

File: HW_5/Task_1.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+let smileBufer = '';
+for (let i = 0; i < 5; i++) {
+    smileBufer = smileBufer + SMILE;
```
Comment (anatoly-karpovich) [inline HW_5/Task_1.js]:
```suggestion
    smileBufer = smileBufer + smile;
```

File: HW_5/Task_3.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+
+  for (let i = 0; i < word.length; i++) {
+    for (let j = 0 ; j < consonants.length; j++) {
+        if (word[i] === consonants[j] ) {
+            consonantsCount++;
+           // console.log(`Буква ${word[i]} согласная ${consonants[j]}`);; 
+        } 
+    }
+  }
```
Comment (anatoly-karpovich) [inline HW_5/Task_3.js]: ух ты, аж 4 цикла! Попробуй vowels.includes(word[i]) и consonants.includes(word[i])

**Review by anatoly-karpovich — APPROVED** [review]

**Review by VladimirLQA — APPROVED** [review]

**Review by anatoly-karpovich — APPROVED** [review]
Красава!

**Pull request merged**